mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -263,7 +263,7 @@ PopulateParamsXDL::initParametersForwardI8[
 };
 // clang-format on
 
-const InitParams PopulateParamsXDL::universalParameters = {32, 64, 4};
+const InitParams PopulateParamsXDL::universalParameters = {32, 64, 4, 4};
```

Comment:
I'm not sure if adding kpack by default to be 4 on xdlops is fair. Also + @manupak / @krzysz00 for their opinions.

Reply:
Yeah, I'm not sure about that either.